CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.6.0
+
+* Add support for optional parameter `is_csv` in `prepare_upload()` file upload utility function. This fixes a bug to ensure that the file is downloaded as a CSV rather than a TXT file.
```

Comment:
```suggestion
* Fix a bug in Send a file by email. Add support for an optional `is_csv` parameter in the `prepare_upload()` utility. This ensures that the file is downloaded as a CSV rather than a TXT file.
```

Reply:
Looks good to me!

Reply:
Thanks Karl. 

Wonder if it should be 'in the `prepare_upload()` function' or just 'in `prepare_upload()`' rather than 'in the `prepare_upload()` utility' though? I don't think the word utility is meaningful

Reply:
That does make better sense to me - I've updated with your initial suggestion there.  Does that sound OK?

Reply:
That's where your subject matter expertise comes in very handy @idavidmcdonald.

'Utility' isn't meaningful to me either, but I wasn't sure if it was a specific term.

Let's go with your suggestion.